s/elements/stickpad/style.css.ts

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
I think it would be best if:
- the `nub-stickpad` has a border that kind of makes it more obvious the area it covers
- the `nubstick` is styled to be kinda transparent

Reply:
i tried reusing nub stick by adding some shadow parts to it and then styling it to make it transparent thats right, do you like this idea ? maybe im missing something, for now it seems like it works ..

Reply:
i think we should have a border around it by default, otherwise developers might get confused when they first put it on their page.

then, developers can choose to disable the border with some simple css:
```cs
nub-stickpad {
  border: none;
}
```